ModelTraining/README.md

```diff
@@ -1,6 +1,29 @@
 # Repo to explain FastAI modeling methodology
 - Author : Aayush Agrawal (aaagraw@microsoft.com)
 
+## Setup
+
+Author: Bruno Grande
+
+The following instructions describe how I was able to install the dependencies for these Jupyter notebooks. I was running into a version conflict with `pip install`. I was able to resolve dependencies using `uv pip install`.
+
+```console
+# Navigate to ModelTraining subdirectory
+cd aifororcas-livesystem/ModelTraining
+
+# Create a new conda environment with Python 3.8
+conda create -n <env-name> python=3.8
+conda activate <env-name>
+
+# Install uv (better at resolving package version conflicts)
+curl -LsSf https://astral.sh/uv/install.sh | sh
+
+# Install dependencies using `uv pip` (instead of plain `pip`)
+uv pip install -r requirements.txt
+```
+
+I'm also including a full list of installed packages and versions in `requirements.lock.txt`, which was generated using `pip freeze`.
```

Comment:
Should requirements.lock.txt be updated every time requirements.txt is updated?
PR #206 and #207 both update requirements.txt

Reply:
I included this file to act as a reference. We'll be adopting some tooling that will allow us to keep an up-to-date lockfile as we install packages (_e.g._ poetry) so it won't have to be manually maintained.